docs/methodology.ipynb

```diff
@@ -0,0 +1,58 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Methodology\n",
+    "\n",
+    "This chapter outlines the comprehensive methodology used to calibrate income distributions across constituencies. Our approach combines multiple data sources and statistical techniques to ensure accurate representation of earnings across different geographical areas and demographic segments. The process consists of six major steps, each building upon the previous to create a robust calibration framework.\n",
```

Comment:
Claude has a natural bias for being a bit self-complimentary. Could you tone it down a bit?